mrtt-ui/src/validation/multiSelectWithOther.js

```diff
@@ -6,23 +6,32 @@ export const otherIsCheckedButInputIsEmptyValidation = (schema) =>
     name: 'isOtherCheckedAndEmpty',
     message: language.multiselectWithOtherFormQuestion.validation.clairfyOther,
     test: (value, context) => {
+      console.log('idotherempty')
       const {
         parent: { otherValue }
       } = context
-      return otherValue !== undefined && otherValue !== ''
+      return otherValue.choice !== undefined && otherValue.choice !== ''
     }
   })
 
 export const multiselectWithOtherValidation = yup.object({
   selectedValues: yup
     .array()
-    .of(yup.string())
+    .default([])
     .when('isOtherChecked', {
-      is: false || undefined,
-      then: (schema) =>
-        schema.min(1, language.multiselectWithOtherFormQuestion.validation.selectAtleastOneItem),
+      is: (value) => {
+        console.log(value)
+        return value === false // doesnt play well with undefined, or !value. We need to explicitly set isOtherChecked to false if undefined.
```

Comment:
this is why we need the .default(false) and also weirdly the .default([]). Wasnt in the docs that it wont interpret an undefined, even if I set it with !!value.  Feels like a bug.